feature/fortune/src/main/java/org/sopt/official/feature/fortune/feature/fortuneAmulet/FortuneAmuletScreen.kt

```diff
@@ -26,52 +26,74 @@ package org.sopt.official.feature.fortune.feature.fortuneAmulet
 
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.PaddingValues
 import androidx.compose.foundation.layout.Spacer
-import androidx.compose.foundation.layout.fillMaxHeight
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.SpanStyle
 import androidx.compose.ui.text.buildAnnotatedString
 import androidx.compose.ui.text.withStyle
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import org.sopt.official.designsystem.SoptTheme
 import org.sopt.official.feature.fortune.component.CircleShapeBorderButton
 import org.sopt.official.feature.fortune.component.UrlImage
 
 @Composable
 internal fun FortuneAmuletRoute(
-    paddingValue: PaddingValues,
     navigateToHome: () -> Unit,
+    viewModel: FortuneAmuletViewModel = hiltViewModel(),
 ) {
-    FortuneAmuletScreen(
-        paddingValue = paddingValue,
-        navigateToHome = navigateToHome
-    )
+    val state by viewModel.state.collectAsStateWithLifecycle()
+
+    when {
+        state.isLoading -> {
+            // Loading View
+        }
+
+        state.isFailure -> {
+            // Error View
+        }
+
+        else -> {
+            FortuneAmuletScreen(
+                description = state.description,
+                name = state.name,
+                imageColor = state.imageColor,
+                imageUrl = state.imageUrl,
+                navigateToHome = navigateToHome
+            )
+        }
+    }
 }
 
 @Composable
 private fun FortuneAmuletScreen(
-    paddingValue: PaddingValues,
+    description: String,
+    name: String,
+    imageColor: Color,
```

Comment:
선생님 이게 뭘까요?

Reply:
```suggestion
  description: @Composable ColumnScope.() -> Unit
```

카드 설명 텍스트를 이렇게 description으로 받아와서 그리는 방법도 있을 것 같아요.

Reply:
확장성을 생각하면 그게 나을 것 같네요!
적용하겠습니다

Reply:
확장성을 생각한 조언 감사합니다!
반영하도록 할게요 :)